db/migrate/20200716101605_create_doorkeeper_tables.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+class CreateDoorkeeperTables < ActiveRecord::Migration[6.0]
+  def change
+    create_table :oauth_applications do |t|
+      t.string  :name,    null: false
```

Comment:
@hoangmirs Do we need this table?

Reply:
Yes, we don't use `oauth_applications` and `oauth_access_grants` currently, let me check and remove them